v3/fleet/control-proxy.service

```diff
@@ -0,0 +1,31 @@
+[Unit]
+Description=run apigateway as control proxy
+Requires=docker.service
+After=docker.service bootstrap.service zk-health.service
+
+[Service]
+Environment="IMAGE=etcdctl get /images/control-proxy"
+Environment="INTERNAL_CONTROL_ELB=etcdctl get /images/control-proxy"
```

Comment:
@chr0n1x @matthewdfuller: This value is in the `/etc/environment` file as CONTROL_ELB.  Can I omit this and us `$($CONTROL_ELB)` in the unit below?